ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -1271,6 +1328,10 @@ func (pg *DEXOnboarding) waitForConfirmationAndListenForBlockNotifications() {
 	asset.RemoveTxAndBlockNotificationListener(DEXOnboardingPageID)
 	asset.AddTxAndBlockNotificationListener(&sharedW.TxAndBlockNotificationListener{
 		OnBlockAttached: func(_ int, _ int32) {
+			if pg.AssetsManager.DEXCInitialized() && !pg.AssetsManager.DexClient().IsLoggedIn() {
+				// Don't update conf if we are not yet logged in.
+				return
+			}
 			pg.bondConfirmationInfo.currentBondConf++
 			pg.ParentWindow().Reload()
```

Comment:
If it is not initialized, confs should be updated? I think just check if it's not logged in, regardless of whether it is initialized or not. Or check `if not initialized OR not logged in`.